idaes/core/util/model_statistics.py

```diff
@@ -1284,13 +1296,45 @@ def large_residuals_set(block, tol=1e-5):
         greater than tol which appear in block
     """
     large_residuals_set = ComponentSet()
+    if return_residual_values:
+        residual_values = dict()
     for c in block.component_data_objects(
             ctype=Constraint, active=True, descend_into=True):
-        if c.active and value(c.lower - c.body()) > tol:
-            large_residuals_set.add(c)
-        elif c.active and value(c.body() - c.upper) > tol:
+            
+        r = 0.0 # residual
+        
+        # check the lower bound
+        # skip if inequality constraint
```

Comment:
This comment is a bit misleading because the logic as-written will still compute residuals for violated bounds on inequality constraints. Maybe just say "skip if no lower bound set"

Reply:
Good suggestion.